scm/driver/harness/pr.go

```diff
@@ -99,44 +99,38 @@ func (s *pullService) Close(context.Context, string, int) (*scm.Response, error)
 // native data structures
 type (
 	pr struct {
-		Author struct {
-			Created     int    `json:"created"`
-			DisplayName string `json:"display_name"`
-			Email       string `json:"email"`
-			ID          int    `json:"id"`
-			Type        string `json:"type"`
-			UID         string `json:"uid"`
-			Updated     int    `json:"updated"`
-		} `json:"author"`
-		Created       int    `json:"created"`
-		Description   string `json:"description"`
-		Edited        int    `json:"edited"`
-		IsDraft       bool   `json:"is_draft"`
-		MergeBaseSha  string `json:"merge_base_sha"`
-		MergeHeadSha  string `json:"merge_head_sha"`
-		MergeStrategy string `json:"merge_strategy"`
-		Merged        int    `json:"merged"`
-		Merger        struct {
-			Created     int    `json:"created"`
-			DisplayName string `json:"display_name"`
-			Email       string `json:"email"`
-			ID          int    `json:"id"`
-			Type        string `json:"type"`
-			UID         string `json:"uid"`
-			Updated     int    `json:"updated"`
-		} `json:"merger"`
-		Number       int    `json:"number"`
+		Author      principal `json:"author"`
+		Created     int64     `json:"created"`
+		Description string    `json:"description"`
+		Edited      int64     `json:"edited"`
+		IsDraft     bool      `json:"is_draft"`
+
+		MergeTargetSHA   *string    `json:"merge_target_sha"`
+		MergeBaseSha     string     `json:"merge_base_sha"`
+		Merged           *int64     `json:"merged"`
```

Comment:
for consistency with other driver implementations you can use `null.Int64` and `null.String`.  example here https://github.com/drone/go-scm/blob/fcc8453fa15d03845c2017d0388d9e1e159d0831/scm/driver/gitlab/org.go#L40